spec/features/test_for_ci_spec.rb

```diff
@@ -7,9 +12,9 @@
     sign_in_with_user(user)
   end
 
-  scenario "the test runs", driver: :headless_firefox do
+  scenario "the test runs", driver: :headless_firefox, broken: true do
```

Comment:
With the name fixed, we could move `driver: :headless_firefox` and `broken: true`) up to line 8 so the whole spec file runs with those options.